pkg/fakerp/server.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/ghodss/yaml"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
```

Comment:
```
gnericapiserver "k8s.io/apiserver/pkg/server"
genericmux "k8s.io/apiserver/pkg/server/mux"
```

Did you checked these ones? They are already in our glide.

Reply:
OR in some sub-packages in k8s they use `"github.com/gorilla/mux` which very similar to what you use here,

Reply:
I was wondering whether we could get away with avoiding the use of an external library. Not sure it's worth it for testing code. @jim-minter thoughts?